packages/opds-web-client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "opds-web-client",
-  "version": "0.1.20",
+  "version": "0.1.21",
```

Comment:
I think I might have used this version number so you might need to update it again.

Reply:
Forgot to pull, thanks.